addons/markers/functions/fnc_isMap.sqf

```diff
@@ -0,0 +1,17 @@
+/*
+ * Author: SzwedzikPL
+ * Checks if item is a map
+ *
+ * Arguments:
+ * 0: Item class <STRING>
+ *
+ * Return Value:
+ * true if given item is a map <BOOL>
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_item"];
+
+getNumber (configFile >> "CfgWeapons" >> _item >> QGVAR(isMap)) > 0
```

Comment:
It should be faster and simpler to use something like:

`(_item find MAPBASE) == 0`